src/apps/eth/eth_verify.c

```diff
@@ -136,13 +137,19 @@ app_eth_sign_error_t app_eth_verify_standard_transaction(const ETHSignRequest* r
     if (params == NULL) {
         return APP_ETH_SIGN_ERR_INVALID_INPUT;
     }
-    if (request->value.size == 0) {
-        // Must transfer non-zero value.
-        return APP_ETH_SIGN_ERR_INVALID_INPUT;
-    }
     if (request->data.size != 0) {
-        // Standard tx has no data.
-        return APP_ETH_SIGN_ERR_INVALID_INPUT;
+        char hex[sizeof(request->data.bytes) * 2 + 1] = {0};
```

Comment:
can also do: `hex[BB_HEX_SIZE(request->data.bytes)]`

Reply:
IMO doesn't really increase legibility, can be done later.